src/templates/product/ProductVariants.js

```diff
@@ -1,4 +1,4 @@
-import React, { Component } from 'react'
+import React, { Component } from 'react/index.js'
 import VariantItem from "./VariantItem.js";
```

Comment:
all absolute imports should come before relative imports.

and use single quotes consistently across the project.